OracleEssbase/dockerfiles/21.1.0/Dockerfile

```diff
@@ -0,0 +1,125 @@
+#
+# Copyright (c) 2021, Oracle and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+#
+
+#
+# Base image
+#
+ARG FMW_INFRA_IMAGE=oracle/fmw-infrastructure:12.2.1.4
+FROM ${FMW_INFRA_IMAGE} as base
+
+# Labels
+# ------
+LABEL "provider"="Oracle"                                             \
+      "issues"="https://github.com/oracle/docker-images/issues"       \
+      "maintainer"="Anil Arora <anil.arora@oracle.com>"               \
```

Comment:
Think about whether you want your personal email published. I usually recommend using an email alias instead.

Reply:
I will update this in a subsequent change to give me some time to get an alias generated.